### PR TITLE
fix/CI-138/fix-change-card-status

### PR DESCRIPTION
**Pull request by julianromero-bc**
## Description
add logic for updating card status change after lock, handle error show when no balances

## Demo
demo-fix-status-change

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/scenes/DebitCard/CardDashboard/FundsBox/FundsBox.model.tsx [hunk 1]
```diff
@@ -49,3 +49,9 @@ export const LoadingDetail = () => (
     </SkeletonLoader>
   </>
 )
+
+export const ErrorState = () => (
+  <Text size='16px' weight={500} color='grey400' capitalize lineHeight='45px'>
+    Failed to load balances
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/scenes/DebitCard/CardDashboard/FundsBox/FundsBox.model.tsx:55]: use Formatted Message

**Review by schnogz — DISMISSED** [review]

File: packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/sagas.ts [hunk 2]
```diff
@@ -43,9 +43,12 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
...
       console.error('Failed to get current card account', errorHandler(e))
       const eligibleAccounts = yield select(selectors.components.debitCard.getEligibleAccounts)
-      yield put(
-        A.getCurrentCardAccountFailure(!isEmpty(eligibleAccounts) ? eligibleAccounts[0] : null)
-      )
+      if (!isEmpty(eligibleAccounts)) {
+        // In case of failure it is set the default account as current
+        yield put(A.getCurrentCardAccountSuccess(eligibleAccounts[0]))
+      } else {
+        yield put(A.getCurrentCardAccountFailure('Could not get user funds'))
```
Comment (awiermanncasas-bc) [inline packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/sagas.ts:50]: Should the `Could not get user funds ` text be translated too?

Comment (julianromero-bc) [reply packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/sagas.ts:50]: Is not being shown to the user

File: packages/blockchain-wallet-v4-frontend/src/scenes/DebitCard/CardDashboard/FundsBox/FundsBox.model.tsx (hunk 1 above)
Comment (julianromero-bc) [reply packages/blockchain-wallet-v4-frontend/src/scenes/DebitCard/CardDashboard/FundsBox/FundsBox.model.tsx]: fixed

**Review by schnogz — APPROVED** [review]

**Pull request merged**